test/ui/cypress/e2e/deployment_driver.cy.js

```diff
@@ -18,38 +11,101 @@ describe('Deployment driver web UI', () => {
     cy.get('@main').contains('Deployment Steps')
     cy.get('@main').get('div.deployProgress').as('statusArea')
     cy.get('@statusArea').get('button.cancelButton').contains('Cancel Deployment')
+
+    cy.get('.pf-c-brand').should('be.visible')
+    cy.get('.pf-c-alert__title')
+      .contains('You currently have a subscription to Ansible Automation Platform')
+    cy.get('p').contains('To manage or setup new subscription, visit the')
+    cy.get('p > .pf-c-button')
+      .contains('Red Hat Hybrid Cloud Console')
+      .should('have.attr', 'href', 'https://console.redhat.com/')
+
+    // Check the H1 title
+    cy.get('h1')
+      .contains('Ansible Automation Platform Deployment Engine')
+      .should('be.visible') 
   })
 
   it('left navigation has expected navigation links', () => {
+    cy.get('#nav-toggle').click()
     cy.get('div#page-sidebar').as('navigation')
+
     cy.get('@navigation').contains('Deployment')
+      .should('be.visible')
+      .should('have.attr','href').and('include','/')
+
     cy.get('@navigation').contains('Documentation')
+      .should('be.visible')
+      .should('have.attr','href').and('include', '/documentation')
+
     cy.get('@navigation').contains('Logout')
+      .should('be.visible')    
   })
 
   it('list of deployment steps contains expected steps', () => {
-    cy.get('div.deploy-step>ul').as('steps')
```

Comment:
This is better implementation of the test because it is easily readable (i.e. easier maintenance). Let's put it back and remove the for loop and the fixture file because they add complexity.

Reply:
Updated